### PR TITLE
doc: add documents

### PR DESCRIPTION
**Pull request by karasusan**
## Add documents
- video streaming
- audio streaming
- data channel

## TODO
- translate in English

### CODE REVIEW

File: Documentation~/en/videostreaming.md [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
+## Codec
 
-## <a id="codec"/> Codec
+There are two types of encoder for video streaming, one is using hardware for encoding and one is using software. Regarding for kind of codecs, the hardware encoder uses `H.264`, and the software encoder uses `VP8`.
 
-ビデオストリーミングで利用するエンコーダーには、ハードウェアで処理するものと、ソフトウェアで処理するものがあります。ハードウェアエンコーダーとしては `H.264` が利用可能です。ソフトウェアエンコーダーを利用する場合は、`VP8` あるいは `VP9` を利用します。
-
-`WebRTC.Initialize` メソッドの引数に `EncoderType` を指定することで、
-ソフトウェアエンコーダーとハードウェアエンコーダーのいずれかを選択することができます。
+Specifying an `EncoderType` for the `WebRTC.Initialize` method argument, it can be selected using software encoder or hardware one.
```
Comment (sindharta) [inline Documentation~/en/videostreaming.md:9]: We can select the type of encoder by specifying `EncoderType` in `WebRTC.Initialize`'s method argument.

File: Documentation~/en/audiostreaming.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Audio Streaming
```
Comment (sindharta) [inline Documentation~/en/audiostreaming.md:1]: This is under en, so should be written in English ?

File: Documentation~/en/datachannel.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Data Channel
```
Comment (sindharta) [inline Documentation~/en/datachannel.md:1]: This is under en, so should be written in English ?

File: Documentation~/en/audiostreaming.md (hunk 2 above)
Comment (karasusan) [reply Documentation~/en/audiostreaming.md:1]: Yes, I will request Chris translating it in English

File: Documentation~/en/datachannel.md (hunk 3 above)
Comment (karasusan) [reply Documentation~/en/datachannel.md:1]: Yes, I will request Chris translating it in English

**Pull request merged**